ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -153,10 +153,10 @@ objects:
 
         i=0
         MAX_RETRIES=10
-        until oc wait "${ROUTER_DEPLOYMENT}" -n "${ROUTER_NAMESPACE}" --for condition=available --timeout=10m || [ $i -eq $MAX_RETRIES ]; do
+        until oc --request-timeout=2m rollout status "${ROUTER_DEPLOYMENT}" -n "${ROUTER_NAMESPACE}" -w || [ $i -eq $MAX_RETRIES ]; do
```

Comment:
I think we should drop the `MAX_RETRIES` check here.  It's originally from fe24e8b2 (#1773), but:

```console
$ i=0
$ MAX_RETRIES=3
$ until false || [ $i -eq $MAX_RETRIES ]; do i=$((i+1)); echo "$i"; [ $i -eq $MAX_RETRIES ] && echo "timeout" && break; done
1
2
3
timeout
$ i=0
$ until false; do i=$((i+1)); echo "$i"; [ $i -eq $MAX_RETRIES ] && echo "timeout" && break; done
$ until false; do i=$((i+1)); echo "$i"; [ $i -eq $MAX_RETRIES ] && echo "timeout" && break; done
1
2
3
timeout
```

You never hit the case where the `MAX_RETRIES` condition in this line is true, because you always hit the `exit 1` timeout inside the loop body first.

CC @abhinavdahiya

Reply:
will drop it - it is redundant.